### PR TITLE
Add progress_bar inclusion_tag

### PR DESCRIPTION
**Pull request by phlax**
as in title

### CODE REVIEW

File: pootle/templates/browser/_progressbar.html [hunk 1]
```diff
@@ -1,13 +1,11 @@
 {% load i18n %}
-<script type="text/template">
-  <span class="legend translated"></span>{% blocktrans %}<span class="value translated">0</span>% translated{% endblocktrans %}<br />
-  <span class="legend fuzzy"></span>{% blocktrans %}<span class="value fuzzy">0</span>% needs work{% endblocktrans %}<br />
-  <span class="legend untranslated"></span>{% blocktrans %}<span class="value untranslated">0</span>% untranslated{% endblocktrans %}
-</script>
-<table class="graph">
+<table class="graph" original-title="
+<span class='legend translated'></span>{% blocktrans %}<span class='value translated'>{{ translated }}</span>% translated{% endblocktrans %}
```
Comment (unho) [inline pootle/templates/browser/_progressbar.html]: Please use double quotes for HTML attributes.

File: pootle/apps/pootle_misc/templatetags/common_tags.py [hunk 2]
```diff
@@ -15,6 +15,19 @@
...
+def progress_bar(total, fuzzy, translated):
+    if not total:
+        return dict(fuzzy=0, translated=0, untranslated=0)
+    untranslated = total - translated - fuzzy
+    return dict(
+        fuzzy=round((float(fuzzy)/total) * 100),
+        translated=round(
+            (float(translated)/total) * 100),
+        untranslated=round(
+            (float(untranslated)/total) * 100))
```
Comment (unho) [inline pootle/apps/pootle_misc/templatetags/common_tags.py]: My only concern would be if this sum up to 100%.

File: pootle/apps/pootle_misc/templatetags/common_tags.py [hunk 3]
```diff
@@ -15,6 +15,19 @@
 register = template.Library()
 
 
+@register.inclusion_tag('browser/_progressbar.html')
+def progress_bar(total, fuzzy, translated):
+    if not total:
+        return dict(fuzzy=0, translated=0, untranslated=0)
```
Comment (unho) [inline pootle/apps/pootle_misc/templatetags/common_tags.py]: Maybe untranslated 100%?

File: pootle/templates/browser/_progressbar.html (hunk 1 above)
Comment (phlax) [inline pootle/templates/browser/_progressbar.html]: this is inserted **into** an html attribute

File: pootle/apps/pootle_misc/templatetags/common_tags.py (hunk 3 above)
Comment (phlax) [inline pootle/apps/pootle_misc/templatetags/common_tags.py]: can you have a 100% of 0 - i think the answer is μ tbh

**Comment by unho** [comment]
lgtm

**Pull request merged**